src/main/kotlin/no/nav/familie/ef/iverksett/tilbakekreving/OpprettTilbakekrevingTask.kt

```diff
@@ -40,17 +40,21 @@ class OpprettTilbakekrevingTask(private val iverksettingRepository: Iverksetting
             logger.warn("OpprettTilbakekrevingTask ikke utført - tilkjentYtelse er null, Behandling: $behandlingId")
             return
         }
-        //val beriketSimuleringsresultat = hentBeriketSimulering(iverksett)
-        val nyIverksett = iverksett//.oppfriskTilbakekreving(beriketSimuleringsresultat)
+
+        val beriketSimuleringsresultat = hentBeriketSimulering(iverksett)
+        val nyIverksett = beriketSimuleringsresultat.fold(
+                { iverksett.oppfriskTilbakekreving(it) },
+                { iverksett }, // bruk eksisterende
+        )
```

Comment:
Burde tasken feile hvis kallet mot tilbakekreving feiler? Sånn att den får rekjört i stedet for att den bruker den eksisterende?

Reply:
Jeg er usikker på hva som blir det riktige. Jeg tenker at det er bedre å få registrert tilbakekreving med det vi har enn å vente og øke risikoen for at det _blir_ endret, og at varselteksten blir mindre relevant.